tidb-cloud/premium/backup-and-restore-premium.md

```diff
@@ -30,49 +30,39 @@ Backup files can originate from the following sources:
 
 ## Automatic backups
 
-TiDB Cloud automatically backs up your instance data, letting you restore data from a backup snapshot to minimize data loss in the event of a disaster.
+{{{ .premium }}} offers enhanced automatic backup capabilities, combining high-frequency snapshots with log backups to maximize data reliability for production environments.
 
-### Learn about the backup setting
+### Automatic backup policies
 
-Compared with {{{ .starter }}} clusters and {{{ .essential }}} clusters, {{{ .premium }}} offers enhanced backup capabilities, including longer retention and hourly backup support.
+{{{ .premium }}} instances provide a comprehensive data protection strategy through a multi-layered backup architecture:
 
-| Backup setting   | {{{ .starter }}} (free) | {{{ .starter }}} (with spending limit > 0) | {{{ .essential }}} | {{{ .premium }}} |
-|------------------|--------------------------|---------------------------------------------|--------------------|------------------|
-| Backup Cycle     | Daily                    | Daily                                       | Daily              | Daily + Hourly   |
-| Backup Retention | 1 day                    | 30 days                                     | 30 days            | Up to 33 days    |
-| Backup Time      | Fixed time               | Configurable                               | Configurable       | Configurable     |
+- Point-in-Time Recovery (PITR)
+    - Retention period: 7 days.
```

Comment:
```suggestion
| Backup type | Retention period | Restore granularity |
| --- | --- | --- |
| **Point-in-time recovery (PITR)** | 7 days | Restore to any specific point in time within the 7-day window. |
| **Hourly snapshot** | 7 days | Restore from any hourly snapshot generated within the last 7 days. |
| **Daily snapshot** | 33 days | Restore from any daily snapshot generated within the last 33 days. By default, daily snapshots are captured at 00:00 UTC. |
```
Replcae L39-49 with a table with a table to make it easier to compare backup type.